concent_api/core/tasks.py

```diff
@@ -58,6 +58,20 @@ def upload_finished(subtask_id: str) -> None:
         if parse_datetime_to_timestamp(subtask.next_deadline) < get_current_utc_timestamp():
             # Worker makes a payment from requestor's deposit just like in the forced acceptance use case.
 
+            update_subtask_state(
```

Comment:
same as above- you moved this block to get `transaction.on_commit(` at the end of function. Is it necessary?

Reply:
Same here, I wanted to make it consistent.